README.rst

```diff
@@ -49,6 +49,88 @@ You can run all of the tox environments using the command ``tox``.
 
 Flocker is also tested using `continuous integration`_.
 
+Cloudformation Deployment
+-------------------------
+
+* Generate a Cloudformation template:
+
+```
+$ python ./admin/installer/cloudformation.py > flocker-cloudformation-template.json
+```
+
+* Validate the template:
+
+```
+$ aws cloudformation validate-template --template-body file://flocker-cloudformation-template.json
+```
+
+* Boot the cluster:
+
+```
+$ cfn \
+    --region us-east-1 \
+    --create flocker-cloudformation-template.json \
+    --parameter KeyName=<aws_ssh_keyname> \
+    --parameter AccessKeyID=<aws_key_id> \
+    --parameter SecretAccessKey=<aws_secret_access_key> \
+    flocker-stack-$RANDOM
```

Comment:
:grey_question: nit: is it useful for the user to know this for later? Might be better served by doing:

```
TMP_ID=$RANDOM
echo $TMP_ID
....
flocker-stack-$TMP_ID
```

Not needed if cfn prints this out again, couldn't verify during review.